### PR TITLE
Update Unit Test Vectors

### PR DESCRIPTION
**Pull request by SmartArray**
### New TestVectors for Unit Test 

This PR updates several test vectors used in unit testing (mostly regtest), such as RegTest block/utxo snapshots and txId snapshots.

They have changed due to:

1. updated fee model (`minRelayFee`)
2. default TX version set to `2`

With these updates, `make check` should yield positive results again.

### CODE REVIEW

**Comment by SmartArray** [comment]
The last error seems to be yielded from `qt/test/test_digibyte-qt`. Which is interesting because I can't reproduce on my machine. It runs through here...

**Comment by SmartArray** [comment]
Awesome work guys! Going to test it out now

**Review by digicontributer — APPROVED** [review]
ACK.

Thanks everyone.

**Review by gto90 — APPROVED** [review]
ACK

Great work @SmartArray and @digicontributer

**Review by JaredTate — APPROVED** [review]
ACK. Great work guys!

So I had a few issues that I ran into with my local dev environment I wanted to pass along in case others run into issues compiling & running 'make check.'

This for macOS Monterey 12.4 with latest Xcode 13.4.1

So after much pain here is what worked for me following multiple issues with my local dev environment. 

1) Fix my infinite apple app store sign-in loop that messed up Xcode update.
2) Remove & reinstall the latest Xcode.
3) Remove legacy bash scripts & get zsh shell sorted w proper paths
4) Get Homebrew happy (doctor, cleanup, upgrade)
5) Manually install boost 1.71 to the cellar after compiling it.
6) Remove all references to OpenSSL on my system and homebrew & make sure the compiler used the osx native LibreSSL 2.8.3.

First I noted that depends always wanted boost 1.71, which was stated here:
https://github.com/DigiByte-Core/digibyte/blob/fix/unit_tests/doc/dependencies.md

Before that, I cleaned up my local dev environment. I wiped Xcode, reinstalled & erased my old bash scripts, and made sure zsh was working correctly as default shell.

Then I:
```
brew upgrade
brew update
brew cleanup 
brew doctor
```

Once brew was happy & zsh were good to go I then set about getting boost 1.71 as system default for brew.

Homebrew only comes with 1.76 or 1.78 as a Formulae, so I compiled 1.71 from scratch and added it manually to brew.

I went here, downloaded boost_1_71_0_rc2.7z & then:
https://boostorg.jfrog.io/ui/native/main/release/1.71.0/source/

```
tar -xzf boost_1_71_0_rc2.tar.gz
cd boost_1_71_0
./bootstrap.sh --prefix=/usr/local/Cellar/boost/1.71.0
./b2
./b2 install
```

And walla, doing a "brew info boost" gives me: 
```
jt@Jareds-MacBook-Pro digibyte % brew info boost
boost: stable 1.78.0 (bottled), HEAD
Collection of portable C++ source libraries
https://www.boost.org/
/usr/local/Cellar/boost/1.71.0 (14,254 files, 154.5MB)
  Built from source
```
After more research I realized the errors I was still getting were due to Openssl. Realizing that since Catalina 10.15 which was being used on our github runners, macOS had removed all Openssl links and used a fork called: LibreSSL 2.8.3

Once I removed all brew links & system references to Open SSL it finished Make Check as expected. 

```
brew uninstall openssl
brew uninstall openssl@3
brew uninstall openssl@1.1
rm -rf /usr/local/etc/openssl
rm -rf /usr/local/etc/openssl@1.1
openssl version
```

So to summarize, install boost 1.71.0... remove all references to OpenSSL in-homebrew, and let OSX Monterey default to LibreSSL 2.8.3 and make check will work without the permissions or other issues.
